app/javascript/hooks/useScript.tsx

```diff
@@ -35,6 +36,7 @@ const useScript = (src) => {
         script.addEventListener("error", setAttributeFromEvent)
       } else {
         // Grab existing script status from attribute and set to state.
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Comment:
suggestion: This entire file is written in a style that predates Typescript, and is used only to facilitate our hacky machine translation process. We can just ignore the entire file by adding [// @ts-nocheck](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#-ts-nocheck-in-typescript-files) to the top of the file/

Reply:
I like this idea but we would also have to attach an eslint ignore for the ts comment itself. What if we just applied eslint rules to the file as needed? So the top of the file would have:
```
/* eslint-disable @typescript-eslint/no-explicit-any */
/* eslint-disable @typescript-eslint/no-unsafe-argument */
```

As an aside, this file could use a thorough refactor to be better in line with Typescript principles.

Reply:
> What if we just applied eslint rules to the file as needed?

Sounds good!

> this file could use a thorough refactor

Might not be worth the time. This file is only used for our current machine translation process, which is on track to be replaced.

Reply:
> Might not be worth the time. This file is only used for our current machine translation process, which is on track to be replaced.

Good point! 

Okay, I will apply the file wide eslint rules. Thanks for the suggestion!

Reply:
fyi this eslint comment can be removed, I see that you removed the other ones already.